backends/graphics/symbiansdl/symbiansdl-graphics.cpp

```diff
@@ -33,7 +33,6 @@ SymbianSdlGraphicsManager::SymbianSdlGraphicsManager(SdlEventSource *sdlEventSou
 
 bool SymbianSdlGraphicsManager::hasFeature(OSystem::Feature f) const {
 	switch (f) {
-	case OSystem::kFeatureFullscreenMode:
```

Comment:
Why did you remove this?

Reply:
On Symbian, the "fullscreen" config option is ignored, and the fullscreen checkbox in the GUI is disabled using `GUI_ONLY_FULLSCREEN`, which suggests that Symbian doesn't actually support fullscreen mode.

Reply:
Indeed! Thanks for the explanation